arch/arm/configs/tsimx6_defconfig

```diff
@@ -965,102 +845,59 @@ CONFIG_MMC_SDHCI_ESDHC_IMX=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
 CONFIG_LEDS_PWM=y
-CONFIG_LEDS_TRIGGER_TIMER=y
-CONFIG_LEDS_TRIGGER_ONESHOT=y
-CONFIG_LEDS_TRIGGER_HEARTBEAT=y
 CONFIG_LEDS_TRIGGER_BACKLIGHT=y
-CONFIG_LEDS_TRIGGER_GPIO=y
 CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_INTF_DEV_UIE_EMUL=y
 CONFIG_RTC_DRV_DS1307=y
 CONFIG_RTC_DRV_ISL12022=y
-CONFIG_RTC_DRV_PCF8563=y
-CONFIG_RTC_DRV_M41T80=y
 CONFIG_DMADEVICES=y
 CONFIG_IMX_SDMA=y
-# CONFIG_MX3_IPU is not set
-CONFIG_DMATEST=m
-CONFIG_UIO=m
-CONFIG_UIO_PDRV_GENIRQ=m
-CONFIG_UIO_DMEM_GENIRQ=m
-CONFIG_UIO_PCI_GENERIC=m
-CONFIG_STAGING=y
+# CONFIG_VIRTIO_MENU is not set
+# CONFIG_VHOST_MENU is not set
 CONFIG_COMMON_CLK_PWM=y
 CONFIG_CLK_IMX8MM=y
 CONFIG_CLK_IMX8MN=y
 CONFIG_CLK_IMX8MP=y
 CONFIG_CLK_IMX8MQ=y
+# CONFIG_IOMMU_SUPPORT is not set
 CONFIG_IMX_GPCV2_PM_DOMAINS=y
-CONFIG_SOC_IMX8M=y
-CONFIG_PM_DEVFREQ=y
-CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=m
-CONFIG_DEVFREQ_GOV_PERFORMANCE=m
-CONFIG_DEVFREQ_GOV_POWERSAVE=m
-CONFIG_DEVFREQ_GOV_PASSIVE=m
-CONFIG_ARM_IMX_BUS_DEVFREQ=m
-CONFIG_PM_DEVFREQ_EVENT=y
-CONFIG_IIO=m
-CONFIG_IIO_BUFFER_DMAENGINE=m
-CONFIG_IIO_BUFFER_HW_CONSUMER=m
-CONFIG_IIO_SW_DEVICE=m
-CONFIG_IIO_SW_TRIGGER=m
-CONFIG_IIO_TRIGGERED_EVENT=m
-CONFIG_IIO_ST_ACCEL_3AXIS=m
-CONFIG_MMA8452=m
-CONFIG_IMX7D_ADC=m
-CONFIG_MCP320X=m
-CONFIG_VF610_ADC=m
-CONFIG_IIO_RESCALE=m
+CONFIG_IIO=y
+CONFIG_MMA8452=y
```

Comment:
If this is optional on any builds, this should probably be left as a module? Reduce bloat on platforms that don't have it.

There are actually a handful of sensors that moved from modules to builtins. I know these have also changed over time, so I think these should all be modules unless there is reason to keep them as builtins.

This applies to multiple configs too it looks like.